requirements.txt

```diff
@@ -25,9 +25,11 @@ hpack==3.0.0
 httptools==0.1.1
 hyperframe==5.2.0
 idna==2.10
+importlib-metadata==1.7.0
```

Comment:
Run `scripts/freeze` in my dev environment, resutling in four more dependencies, `importlib-metadata`, `jeepney`, `SecretStorage` and `zipp`, I don't know why…